docker/package/model.py

```diff
@@ -117,6 +118,9 @@ def gen_spec_systemd_part(package):
                 + f"%{{buildroot}}/%{{_sysconfdir}}/default/{service_name}\n"
             )
             default_files += f"%{{_sysconfdir}}/default/{service_name}\n"
+        if systemd_unit.service_file.service.environment_files is not None:
+            for environment_file in systemd_unit.service_file.service.environment_files:
+                environment_files += f"{environment_file}\n"
```

Comment:
It's wrong way to install them, since it fails with fair error 
```
RPM build errors:
    File not found: /tezos-packaging/rpmbuild/BUILDROOT/tezos-baker-PtKathma-15.0rc1-3.x86_64/etc/default/tezos-baking-%
```
I came up with such proto solution, problem here is that we need to know `networks`, which is defined in `packages.py`, which depends on `model.py`
```suggestion
            environment_files += "\n".join(
                sum(
                    [[x.replace("%i", y) for y in package.networks]
                     for x in systemd_unit.service_file.service.environment_files],
                    []
                )
            )
```

Reply:
it turns out that this ins't actually necessary at all and is already covered by the `default_files` just above.

I removed it and left a comment with the reason why in 165ec1fbea7395a4e094aa8fc51eadbe0c41ebf8